config/environments/production.rb

```diff
@@ -1,5 +1,11 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
+  config.action_mailer.smtp_settings = {
+      :address   => 'smtp.mandrillapp.com',
+      :port      => 587,
+      :user_name => ENV['MANDRILL_USERNAME'],
+      :password  => ENV['MANDRILL_PASSWORD']
+  }
```

Comment:
@vlasiak, please consider the following suggestions:
- this code is duplicated with development configuration file. Learn DRY principle:

> Every piece of knowledge must have a single, unambiguous, authoritative representation within a system.

See the details [here](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and [here](http://c2.com
/cgi/wiki?DontRepeatYourself). 

- Let's move it out from env files into `application.rb`.